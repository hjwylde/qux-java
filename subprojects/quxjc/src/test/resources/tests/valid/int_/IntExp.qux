void main([str] args):
    print 0 ** 0
    print 0 ** 1
    print 1 ** 0
    print 1 ** 1
    print 5 ** 2
    print 2 ** 2
    print -3 ** 2

