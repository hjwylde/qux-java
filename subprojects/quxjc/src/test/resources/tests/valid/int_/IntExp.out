1
0
1
1
25
4
9
